routes/friends.js

```diff
@@ -25,6 +25,7 @@ router.get('/invitations', ensureAuthenticated, (req, res) => {
             const pendingInvitations = doc.map((user) => ({
               id: user._id,
               username: user.username,
+              color: user.color ? user.color : 'warm',
```

Comment:
you could also just say `user.color || 'warm'` (this applied to other locations in the code)

Reply:
oh, hahaha, yeah that's way more consise